momepy/functional/_intensity.py

```diff
@@ -111,3 +111,65 @@ def _calc_nodedensity(group, edges):
         summation_values = pd.Series(np.ones(nodes.shape[0]), index=nodes.index)
 
     return graph.apply(summation_values, _calc_nodedensity, edges=edges)
+
+
+def count(
+    left: GeoDataFrame,
+    right: GeoDataFrame,
```

Comment:
Do we actually need `right` at all? Doesn't the grouping have all the info we need?